spec/descriptions/createGroupMapping.md

```diff
@@ -0,0 +1,7 @@
+Creates a mapping between a group from the IdP (LDAP, OIDC, SAML) and an Instana group.
+
+If the IdP is configured and mappings are enabled, every time a user logs in the `key` `value` pairs for this user sent by the idp will be evaluated.
```

Comment:
Just juggling the syntax, without changing the words to make it easy to read:

`If the IdP is configured and mappings are enabled, the `key` `value` pairs for a user sent by the idp will be evaluated every time this user logs in.`